airflow/decorators/base.py

```diff
@@ -431,18 +431,29 @@ def _expand(self, expand_input: ExpandInput, *, strict: bool) -> XComArg:
         dag = task_kwargs.pop("dag", None) or DagContext.get_current_dag()
         task_group = task_kwargs.pop("task_group", None) or TaskGroupContext.get_current_task_group(dag)
 
-        partial_kwargs, partial_params = get_merged_defaults(
+        default_args, partial_params = get_merged_defaults(
             dag=dag,
             task_group=task_group,
             task_params=task_kwargs.pop("params", None),
             task_default_args=task_kwargs.pop("default_args", None),
         )
-        partial_kwargs.update(
-            task_kwargs,
-            is_setup=self.is_setup,
-            is_teardown=self.is_teardown,
-            on_failure_fail_dagrun=self.on_failure_fail_dagrun,
-        )
+        partial_kwargs: dict[str, Any] = {
+            "is_setup": self.is_setup,
+            "is_teardown": self.is_teardown,
+            "on_failure_fail_dagrun": self.on_failure_fail_dagrun,
+        }
+        base_signature = inspect.signature(BaseOperator)
+        ignore = {
+            "default_args",
+            "kwargs",
+            "do_xcom_push",
+            "multiple_outputs",
+            "params",
+            "task_concurrency",
+        }
```

Comment:
Let’s add some comments to explain why each one is ignored.

Reply:
First, as I mentioned earlier, here's why I thought we should remove `default_args`, `kwargs`, `multiple_outputs`, and `params`.
- `default_args`: This is the target we're working on right now.
- `kwargs`: A common name for a keyword argument.
- `multiple_outputs`: We use `self.multiple_outputs` in subsequent code.
- `params`: Already managed by `partial_params`.

and
- `task_concurrency`: is deprecated(will be replaced by `max_active_tis_per_dag`)
- `do_xcom_push`: I'm actually not sure about `do_xcom_push` yet.

Reply:
Yeah I meant adding the information to the code base so readers don’t need to come back here for the context.

Reply:
I'm still not quite sure about `do_xcom_push`, so if you can tell me what comments I should add to that, I'll add them to the commit.

Reply:
I don't quite understand, but should I comment "in the same boat as `multiple_outputs`"?

Reply:
I added it for now.